recipes/sra-tools/meta.yaml

```
@@ -10,33 +10,34 @@ package:
 source:
   - url: https://github.com/ncbi/sra-tools/archive/{{version}}.tar.gz
     sha256: 10ac0a4d1fafc274bc107de811891d3e803d0713a247581dece4448231883810
+    patches:
+      - 0001-No-libhdf5-libxml2-static-link-sra-tools.patch
     folder: sra-tools
   - url: https://github.com/ncbi/ncbi-vdb/archive/{{vdb_version}}.tar.gz
     sha256: 9a65e3885b9ae1ebecbec871f04ce3162ac3764fb556ecdc8c1e61993e2164aa
+    patches:
+      - 0002-No-libhdf5-libxml2-static-link-ncbi-vdb.patch
     folder: ncbi-vdb
-#  - url: https://github.com/ncbi/ngs/archive/{{ngs_version}}.tar.gz
-#    sha256: 5fde50784760c00b403c2cc42ead15a4e9477697ee439f0a16edb4de3f52dfcc
-#    folder: ngs
 
 build:
   number: 0
```

```suggestion
  number: 1
```
Whoops, I forgot a build number bump :). Before fixing this, I'm letting the CI run through once.

---

Thanks for your help @mbargull! Looks like we're still getting `SERR 2021-04-23T15:29:58 fastq-dump.2.11.0 sys: mbedtls_ssl_get_verify_result returned 0x8 (  !! The certificate is not correctly signed by the trusted CA  )`